.github/workflows/easy-cp.yml

```diff
@@ -0,0 +1,81 @@
+# Copyright 2023 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+name: Cherry-pick Labeled PR to Release Branch
+
+on:
+  pull_request:
+    branches: master
+    types: [labeled]
+
+permissions: write-all
+
+jobs:
+  cherrypick_to_release:
+    name: cherrypick_to_release
+    runs-on: ubuntu-latest
+    if: |
+      (github.event.label.name == format('cp{0} beta', ':') || github.event.label.name == format('cp{0} stable', ':')) &&
+      (github.event.pull_request.merged == true)
+    steps:
+      - name: Get Release Channel
+        run: |
+          echo "CHANNEL=$(echo ${{ github.event.label.name }} | cut -d ':' -f 2 | xargs)" >> $GITHUB_ENV
+      - name: Get Release Candidate Branch
+        run: |
+          curl -o version.json https://raw.githubusercontent.com/flutter/flutter/$CHANNEL/bin/internal/release-candidate-branch.version
+          echo "RELEASE_BRANCH=$(cat version.json | tr -d '\n')" >> $GITHUB_ENV
+          rm version.json
+      - name: Get Cherry Pick PR
+        run: |
+          echo "COMMIT_SHA=$(echo ${{ github.event.pull_request.merge_commit_sha }})" >> $GITHUB_ENV
+      - name: Checkout Flutter Repo
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
+        with:
+          repository: flutter/flutter
+          token: ${{ github.token }}
+          path: flutter
+          ref: master
+          fetch-depth: 0
+      # use same name when checking out branch, since the marketplace action does a hard reset.
+      - name: Attempt CP
+        id: attempt-cp
+        working-directory: ./flutter
+        run: |
+          git config user.name "GitHub Actions Bot"
+          git config user.email "<>"
+          git fetch origin $RELEASE_BRANCH
+          git fetch origin master
+          git checkout -b cp-$COMMIT_SHA --track origin/$RELEASE_BRANCH
+          git cherry-pick $COMMIT_SHA
+          git status
+      # TODO(xilaizhang): move cp template to flutter/flutter.
+      # The Pull Request cherry pick template should be under flutter/.github/PULL_REQUEST_CP_TEMPLATE.md.
+      - name: Get CP Template
```

Comment:
Is this needed if below we use "body-path" to use the template?

Reply:
Yeah i added a TODO here but we can only remove this part after https://github.com/flutter/flutter/pull/139590 lands (or when Kevin is back).

Without an existing template in flutter/flutter, we will have to download the template from somewhere and include it in the opened pull request.

Reply:
With https://github.com/flutter/flutter/pull/139590 ready, can we remove the TODO?

Reply:
Tested around and realized I cannot remove this step because the template isn't on release branches yet.

Currently the template is only on master head and not on release branches. When we are creating the pull request from the release branch, this file path becomes unavailable to us. For now, I still have to download it from the flutter/flutter repo.